internal/integration/ogent/ent/ogent/ogent.go

```diff
@@ -29,6 +34,245 @@ func rawError(err error) jx.Raw {
 	return e.Bytes()
 }
 
+type Boolean bool
+
+func (b *Boolean) Capture(values []string) error {
+	*b = values[0] == "TRUE"
+	return nil
+}
```

Comment:
That whole part will get removed if you run `go generate`, won't it? I guess it has to be part of the `template/handler.tmpl`.

Reply:
Sorry, I forgot to add the new template to add the parsing logic. I committed today and add it to the `template/handler.tmpl`